### DISCUSSION TITLE
Recent update broke Markdown highlighting on 0.9.5, does nvim-treesitter target nvim nightly?

### DISCUSSION THREAD

**Discussion by adrian5** [nvim-treesitter/nvim-treesitter#5931] [Q&A] [answered]
After doing a plugin update today, my treesitter Markdown highlights are broken. Do I need to use nvim `0.10` development releases to ensure future functionality of nvim-treesitter?

**Comment by clason** [comment, marked as answer]
No, you need to follow the Breaking Changes issue.

**Reply by adrian5** [reply]
Thanks, the [last post](https://github.com/nvim-treesitter/nvim-treesitter/issues/2293#issuecomment-1900679583) explains the issues I'm having.
